pages/acknowledgments.tex

```diff
@@ -26,62 +26,64 @@ \section*{How to cite this rulebook}
 \url{https://athome.robocup.org/rules/\YEAR_rulebook.pdf}, \YEAR.
 
 \begin{center}
-\begin{minipage}{0.8\textwidth}
-	\footnotesize%
-	\verbatiminput{citation.bib}
-\end{minipage}
+	\begin{minipage}{0.8\textwidth}
+		\footnotesize%
+		\verbatiminput{citation.bib}
+	\end{minipage}
 \end{center}
 
 \section*{Acknowledgments}
 \label{sec:acknowledgments}
 We would like to thank the members of the Technical Committee who put up the rules and the Organizing Committee who organizes the competition.
 People that have been working on this rulebook as members of one of the league's committees (in alphabetical order):
 \begin{center}
-    \begin{minipage}{0.8\textwidth}
-        \begin{multicols}{3}%
-            \footnotesize
-            \noindent%
-
-            Alex Mitrevski\\
-            Alexander Moriarty\\
-            Caleb Rascon\\
-            Fagner Pimentel\\
-            \columnbreak
-            Johanner Kumert\\
-            Justin Hart\\
-            Katarzyna Pasternak\\
-            Komei Sugiura\\
-            Luca Iocchi\\
-            \columnbreak
-            Mauricio Matamoros\\
-            Matteo Leonetti\\
-            Maxime St-Pierre \\
-            Sammy Pfeiffer\\
-            Sven Wachsmuth\\
-            Tijn van der Zant\\
-        \end{multicols}
-    \end{minipage}
+	\begin{minipage}{0.8\textwidth}
+		\begin{multicols}{3}%
+			\footnotesize
+			\noindent%
+			Adam Golding\\
+			Akinobu Mizutani\\
+			Alexander Moriarty\\
+			Fagner Pimentel\\
+			Hiroyuki Okada\\
+			Florian Lier\\
+			Johannes Kummert\\
+			Justin Hart\\
+			Katarzyna Pasternak\\
+			Luca Iocchi\\
+			Luca Lach\\
+			Luis Contreras\\
+			Mauricio Matamoros\\
+			Matteo Leonetti\\
+			Maxime St-Pierre\\
+			Peter van Dooren\\
+			Sven Wachsmuth\\
+			Yuma Yoshimoto
+		\end{multicols}
+	\end{minipage}
 \end{center}
 
 \noindent We would also like to thank all the people who contributed to the RoboCup@Home league with their feedback and comments.
 People that have been working on this rulebook as members of the league (in alphabetical order):
 \begin{center}
-    \begin{minipage}{0.8\textwidth}
-        \begin{multicols}{2}%
-            \footnotesize
-            \noindent%
-            Florian Lier (@warp1337)\\
-            Hiroyuki Okada (@okadahiroyuki)\\
-            Mark Finean (@mfinean)\\
-            Lars Janssen (@LarsJanssenTUE)\\
-            \columnbreak%
-            Loy van Beek (@LoyVanBeek)\\
-            Matthijs van der Burgh (@MatthijsBurgh)\\
-            Raphael Memmesheimer (@airglow)\\
-            Sebastian Meyer zu Borgsen (@semeyerz)\\
-            Syed Ali Raza (@syedaraza)
-        \end{multicols}
-    \end{minipage}
+	\begin{minipage}{0.8\textwidth}
+		\begin{multicols}{2}%
+			\footnotesize
+			\noindent%
+			Alex Mitrevski\\
+			Caleb Rascon\\
+			Komei Sugiura\\
+			Mark Finean\\
+			Lars Janssen\\
+			Loy van Beek\\
```

Comment:
Have been TC some years back. I still get the notifications :-)

Reply:
Well I put you in the "people who contributed to the RoboCup@Home league with their feedback and comments" section. I think that definitely still fits you.

Reply:
Oh I just realized I totally misread the wording of the first section. I thought it said current members... 
will update

Reply:
IMHO this undervalues the work of previous TC members; being a TC member and actively working on the rulebook is obviously more work than "just" providing feedback and comments. Maybe you could add sections for "current TC members" and "previous TC members" if this distinction is important?

Reply:
To me it is not important. I would just move all former members back up in the first section.

Reply:
Sorry for the nitpick, there's more important stuff indeed. Thnx though